rnd/autogpt_server/test/executor/test_manager.py

```diff
@@ -109,6 +110,7 @@ async def test_agent_execution(server: SpinTestServer):
         test_user,
         data,
         4,
+        200,
```

Comment:
I chose 200 out of nothing. @majdyz @Pwuts, what's a reasonable value here? I was inconsistently failing across some oses, so passed this down

Reply:
What is that parameter? The execution timeout? It defaults to 20 which I think is reasonable. If it can't do a graph execution in 20 seconds we have other problems to address.

Reply:
okay so great news, we can't do it in 20 seconds and that's why the CI kept failing

Reply:
It intermittently fails in this pr because of this code change

Reply:
The blocks you are changing is not used in this test. How is it possible?

Reply:
I have no idea, but if you take this out it fails in CI

Reply:
fails at 30 seconds on windows. Trying 31